packages/nitro-protocol/src/contract/channel.ts

```diff
@@ -8,12 +8,19 @@ export interface Channel {
   chainId: Uint256;
 }
 
+export function isExternalDestination(bytes32: Bytes32): boolean {
```

Comment:
I think you can do something like `: bytes32 is Destination {` to return a now-typed `Destination` object